doc/source/how_to/orchestration/add_pip_dependencies_to_electron.ipynb

```diff
@@ -0,0 +1,132 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "445760b7-70d8-4109-b200-5478b0413337",
+   "metadata": {},
+   "source": [
+    "## How to add pip dependencies to an electron\n",
+    "\n",
+    "In this section, we illustrate the steps in adding a list of PyPI packages that are required to execute an electron.\n",
+    "\n",
+    "First, we import the `DepsPip` class:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "698a93a4-7fbc-4f94-a31b-ea6a3cfb28cc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import covalent as ct\n",
+    "from covalent import DepsPip\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d2f3d5c-6c1c-4103-9e2d-ac4594f329a8",
+   "metadata": {},
+   "source": [
+    "We then pass the list of packages to `DepsPip()`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "3d8064f0-e732-4dac-b97a-77ff19689157",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@ct.electron(\n",
+    "    deps_pip=DepsPip(packages=[\"numpy==0.23\", \"qiskit\"]),\n",
+    ")\n",
+    "def identity(x):\n",
+    "    return x"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7983772f-b510-42be-807a-7ecda42f3ba9",
+   "metadata": {},
+   "source": [
+    "Alternatively, we can specify the path to a `requirements.txt` file that contains the list of required packages. Assuming we require packages in `doc/requirements.txt`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "cac2c232-7e3c-44b5-8242-91d8d6e13882",
+   "metadata": {},
+   "outputs": [
+    {
```

Comment:
@Emmanuel289 we might want to remove this error output

Reply:
@AlejandroEsquivel  Fixed